crates/colorchoice-clap/src/lib.rs

```diff
@@ -36,66 +38,6 @@ impl Color {
     }
 }
 
-/// Argument value for when to color output
-#[derive(Copy, Clone, Debug, PartialEq, Eq)]
-pub enum ColorChoice {
-    Auto,
-    Always,
-    Never,
-}
-
-impl ColorChoice {
-    /// Report all `possible_values`
-    pub fn possible_values() -> impl Iterator<Item = clap::builder::PossibleValue> {
-        use clap::ValueEnum;
-        Self::value_variants()
-            .iter()
-            .filter_map(clap::ValueEnum::to_possible_value)
-    }
-
-    fn as_str(&self) -> &'static str {
-        match self {
-            Self::Auto => AUTO,
-            Self::Always => ALWAYS,
-            Self::Never => NEVER,
-        }
-    }
-}
-
-impl Default for ColorChoice {
-    fn default() -> Self {
-        Self::Auto
-    }
-}
-
-impl std::fmt::Display for ColorChoice {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        self.as_str().fmt(f)
-    }
-}
-
-impl std::str::FromStr for ColorChoice {
-    type Err = String;
-
-    fn from_str(s: &str) -> Result<Self, Self::Err> {
-        <Self as clap::ValueEnum>::from_str(s, false)
-    }
-}
-
-impl clap::ValueEnum for ColorChoice {
-    fn value_variants<'a>() -> &'a [Self] {
-        &[Self::Auto, Self::Always, Self::Never]
-    }
-
-    fn to_possible_value(&self) -> Option<clap::builder::PossibleValue> {
-        Some(clap::builder::PossibleValue::new(self.as_str()))
-    }
-}
```

Comment:
Should we change clap from providing `PossibleValue::help` to not?

This forces anyone using this library into using next-line-help which can get a bit verbose.  I think I modeled `PossibleValue::help`  off of `rg` but I think the descriptions, without `AlwaysAnsi`, are likely obvious enough making it so this isn't pulling its weight.  I checked `git` and their "long help" (man page) doesn't seems to describe each value.

Reply:
> Should we change clap from providing PossibleValue::help to not?

I think clap should still provide the help, but I agree that we can shorten it.

Reply:
So long as its present, it makes it a lot longer than it would otherwise (vertically)

Reply:
I am sorry but I don't understand your comment Can you please expand?

Reply:
```
          Possible values:
          - auto:   Use colored output if writing to a terminal/TTY
          - always: Always use colored output
          - never:  Never use colored output
```
takes up a lot more space than
```
[possible values: auto, always, never]
```